pyproject.toml

```diff
@@ -0,0 +1,54 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+    "setuptools>=60",
+    "setuptools-scm>=8.0"
+    ]
+
+[project]
+name = "aiu-fms-testing-utils"
+dynamic = ["version"]
+description = "Spyre FMS Testing Utils"
+authors = [
+  {name = "Thara Palanivel", email = "tpalan@ibm.com"},
+  {name = "Joshua Rosenkranz", email = "jmrosenk@us.ibm.com"},
+  {name = "Antoni Viros i Martin", email = "aviros@ibm.com"},
+]
+license = "Apache-2.0"
+readme = "README.md"
+requires-python = "~=3.12"
```

Comment:
@JRosenkranz @ani300 @tharapalanivel I also notice the [fms-mo](https://github.com/foundation-model-stack/fms-model-optimizer/blob/9ee258bb638b9884c9dad00ddbbc7b888779bda6/pyproject.toml#L10) has
requires-python = ">3.9,<3.13"
Question is should we relax this or is 3.12 runtime absolutely requirement for this package.

Reply:
done - thanks!